runelite-client/src/main/java/net/runelite/client/plugins/worldmap/RareTreeLocation.java

```diff
@@ -208,7 +209,23 @@ enum RareTreeLocation
 
 	MAPLE_YEW("Maple/Yew trees", 60,
 		// Feldip Hills
-		new WorldPoint(2476, 2893, 0)),
+		new WorldPoint(2476, 2893, 0),
+
+		// Varlamore
+		new WorldPoint(1379, 2876, 0)),
+
+	OAK_MAPLE("Oak/Maple trees", 45,
+		// Varlamore
+		new WorldPoint(1362, 2862, 0),
+		new WorldPoint(1398, 2856, 0)),
+
+	OAK_WILLOW_YEW("Oak/Willow/Yew", 60,
+		// Varlamore
+		new WorldPoint(1412, 2992, 0)),
```

Comment:
I think this is better off just being "Yew", seeing as the icon is centered between three yew trees.